### PR TITLE
Add JS Article Break

### PR DESCRIPTION
**Pull request by dakshshah96**
Refer #1068.

Add JS article for **break**, a part of **Statements and declarations**.

### CODE REVIEW

File: JS-Break.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    if (i == 3) {
+      break;
+    }
+    i += 1;
+  }
+
+  return i * x;
+}
+```
+
```
Comment (atjonathan) [inline JS-Break.md]: Add [repl](https://repl.it) snippet :rocket:

File: JS-Break.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+In the following example, the counter is set up to count from 1 to 99; however, the **break** statement terminates the loop after 14 counts.
+
+```javascript
+for (var i = 1; i < 100; i++) {
+  if (i == 15) {
+    break;
+  }
+}
+```
```
Comment (atjonathan) [inline JS-Break.md]: Add [repl](https://repl.it) snippet :rocket:

**Comment by dakshshah96** [comment]
@atjonathan repl links added.

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Pull request merged**